aten/src/ATen/native/quantized/cpu/qnnpack/include/conv_utils.h

```diff
@@ -8,9 +8,30 @@
 #include <qnnpack/log.h>
 #include <qnnpack/operator.h>
 
+// TODO: Might combine the conv and deconv parameters/
+
 namespace qnnpack {
+
+namespace {
+static inline size_t compute_output_dimension(
+    size_t input_dim,       // Input dimension
+    size_t pad_dim,         // Input padding
+    size_t adjustment_dim,  // Adjustment to the output dimension
```

Comment:
What is adjustment_dim? Note that there is already conv util that can help you determine output shape here: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/ConvUtils.h. If possible let's use that?

Reply:
It doesn't have the deconv output shape.

Reply:
Sounds good.